prebuilt/common/bin/bootpwn

```diff
@@ -16,23 +16,21 @@ unset LD_PRELOAD
 
 echo "[!] Welcome to the Pwn zone. Get your game face on."
 
-/system/xbin/busybox stty columns 85
-/system/xbin/busybox stty rows 46
```

Comment:
It isn't included in either toolbox or toybox, so it doesn't exist on system right now. It might make things cleaner, but it's not a blocker for baking busybox out.